src/redux/store.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksReducer from './books/booksSlice';
+
+const store = configureStore({
+  reducer: {
+    book: booksReducer,
+  },
+});
```

Comment:
- You have done great setting up the redux store and reducers :+1: :+1: But there are some requirements you still need to implement before moving to the next milestone. :+1: :100: 
- [ ] Currently, you only imported the booksReducer to your store, you are meant to also import the `categoriesReducer` here and add it into the redux store. I'll suggest you import the categories reducer and implement that too. :+1: :wave: :smile: 
````jsx
const store = configureStore({
  reducer: { books: booksReducer, categories: categoriesReducer },
})
````